recipes/multiformats-config/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "multiformats-config" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/multiformats-config-{{ version }}.tar.gz
+  sha256: 7eaa80ef5d9c5ee9b86612d21f93a087c4a655cbcb68960457e61adbc62b47a7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=45
+    - wheel
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >={{ python_min }}
+
+test:
+  imports:
+    - multiformats_config
+  commands:
+    - pip install multiformats
```

Comment:
This is not allowed in conda-forge. You must package that too.

```suggestion
```

Reply:
The problem is, that the packages have a circular dependency. We already tried to solve this in an earlier PR [https://github.com/conda-forge/staged-recipes/pull/28404] which failed for other technical issues.

Is there an official way how to handle such an issue in conda-forge?